app/views/games/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="text-center">
   <br>
-  <%= @game.name %>
+  <!-- <%= @game.name %> -->

```

Comment:
@phearn looks like `@game.name` has been commented out and we're displaying `@game.game_status`.  This will should also be addressed in the list games available story after this PR.